site/en/blog/timer-throttling/index.md

```diff
@@ -0,0 +1,238 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Timer throttling
```

Comment:
Based off my own usage of docs, I often decide whether or not to read a post based off the title alone. So I tend towards slightly more verbose titles so that readers get a good idea of what's inside.

Reply:
Are you happy with the URL remaining `timer-throttling`?

Reply:
lol sorry, I just saw your other comment